test/Microsoft.OpenAPI.OData.Reader.Tests/EdmModelOpenApiExtensionsTest.cs

```diff
@@ -31,92 +31,160 @@ public void ConvertToOpenApiThrowsArgumentNullModel()
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void BasicEdmModelToOpenApiYamlWorks(bool isOpenApiV2Spec)
         {
             // Arrange
             IEdmModel model = EdmModelHelper.BasicEdmModel;
+            var openApiConvertSettings = new OpenApiConvertSettings();
+            openApiConvertSettings.OpenApiSpecVersion = isOpenApiV2Spec ? OpenApiSpecVersion.OpenApi2_0 : OpenApiSpecVersion.OpenApi3_0;
 
             // Act
-            string yaml = WriteEdmModelAsOpenApi(model, OpenApiFormat.Yaml);
+            string yaml = WriteEdmModelAsOpenApi(model, OpenApiFormat.Yaml, openApiConvertSettings);
             _output.WriteLine(yaml);
 
             // Assert
-            Assert.Equal(Resources.GetString("Basic.OpenApi.yaml").ChangeLineBreaks(), yaml);
+            if (isOpenApiV2Spec)
+            {
+                Assert.Equal(Resources.GetString("Basic.OpenApi.V2.yaml").ChangeLineBreaks(), yaml);
+            }
+            else
+            {
+                Assert.Equal(Resources.GetString("Basic.OpenApi.yaml").ChangeLineBreaks(), yaml);
+            }
         }
 
-        [Fact]
-        public void MultipleSchemasEdmModelToOpenApiJsonWorks()
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void MultipleSchemasEdmModelToOpenApiJsonWorks(bool isOpenApiV2Spec)
         {
             // Arrange
             IEdmModel model = EdmModelHelper.MultipleSchemasEdmModel;
+            var openApiConvertSettings = new OpenApiConvertSettings();
+            openApiConvertSettings.OpenApiSpecVersion = isOpenApiV2Spec ? OpenApiSpecVersion.OpenApi2_0 : OpenApiSpecVersion.OpenApi3_0;
 
             // Act
-            string json = WriteEdmModelAsOpenApi(model, OpenApiFormat.Json);
+            string json = WriteEdmModelAsOpenApi(model, OpenApiFormat.Json, openApiConvertSettings);
             _output.WriteLine(json);
 
             // Assert
-            Assert.Equal(Resources.GetString("Multiple.Schema.OpenApi.json").ChangeLineBreaks(), json);
+            if (isOpenApiV2Spec)
+            {
+                Assert.Equal(Resources.GetString("Multiple.Schema.OpenApi.V2.json").ChangeLineBreaks(), json);
+            }
+            else
+            {
+                Assert.Equal(Resources.GetString("Multiple.Schema.OpenApi.json").ChangeLineBreaks(), json);
+            }
         }
 
-        [Fact]
-        public void MultipleSchemasEdmModelToOpenApiYamlWorks()
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void MultipleSchemasEdmModelToOpenApiYamlWorks(bool isOpenApiV2Spec)
         {
             // Arrange
             IEdmModel model = EdmModelHelper.MultipleSchemasEdmModel;
+            var openApiConvertSettings = new OpenApiConvertSettings();
+            openApiConvertSettings.OpenApiSpecVersion = isOpenApiV2Spec ? OpenApiSpecVersion.OpenApi2_0 : OpenApiSpecVersion.OpenApi3_0;
```

Comment:
Will make this change. 
Had thought to establish a patter of using boolean value here in case additional filtering was needed would need to explicitly check version, but likely not a substantive gain from that approach.